scripts/test-triton.sh

```diff
@@ -288,6 +293,24 @@ run_benchmark_attention() {
     python $TRITON_PROJ/benchmarks/triton_kernels_benchmark/flash_attention_fwd_benchmark.py
 }
 
+run_benchmarks() {
+  cd $TRITON_PROJ/benchmarks
+  python setup.py install
+  for file in $TRITON_PROJ/benchmarks/triton_kernels_benchmark/*.py; do
+    if ! [ -f "$file" ]; then
+      continue
+    fi
+    benchmark=$(basename -- "$file")
+    if [[ $benchmark = @("__init__.py"|"benchmark_driver.py"|"benchmark_testing.py") ]]; then
+      continue
+    fi
+    echo
+    echo "****** Running ${benchmark%.py} ******"
+    echo
+    python $TRITON_PROJ/benchmarks/triton_kernels_benchmark/$benchmark
```

Comment:
Does this look simpler?
```suggestion
    python $file
```

Reply:
good idea.